src/main/kotlin/com/openlattice/transporter/types/TransporterDatastore.kt

```diff
@@ -0,0 +1,199 @@
+package com.openlattice.transporter.types
+
+import com.geekbeast.configuration.postgres.PostgresConfiguration
+import com.kryptnostic.rhizome.configuration.RhizomeConfiguration
+import com.openlattice.ApiUtil
+import com.openlattice.assembler.AssemblerConfiguration
+import com.openlattice.assembler.PostgresDatabases
+import com.openlattice.postgres.external.ExternalDatabaseConnectionManager
+import com.zaxxer.hikari.HikariDataSource
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Component
+import java.util.*
+
+/**
+ * TransporterDatastore configures entries in [rhizome] postgres configuration
+ */
+@Component
+class TransporterDatastore(
+        private val assemblerConfiguration: AssemblerConfiguration,
+        rhizome: RhizomeConfiguration,
+        private val exConnMan: ExternalDatabaseConnectionManager
+) {
+    companion object {
+        private val logger = LoggerFactory.getLogger(TransporterDatastore::class.java)
+        private val PAT = Regex("""([\w:]+)://([\w_.]*):(\d+)/(\w+)""")
```

Comment:
wat

Reply:
Moved the comment up to clarify this one

Reply:
ah awesome thanks!